cmd/settings/settings_edit.go

```diff
@@ -88,7 +95,7 @@ func RunSettingsEditCommand(args *EditArgs) di.CmdWithApiFn {
 			}
 		}
 
-		nodes, resp, err := doImport(cli.Context, api, string(c), "", args.UnlockedStrategy, args.Timeout)
+		nodes, resp, err := doImport(cli.Context, api, string(c), "", args.UnlockedStrategy, args.LockedStrategy, args.Timeout)
```

Comment:
Doesn't this make the `UnlockNodes` on line 85 obsolete? That looks like a hack to work-around the missing LockedStrategy argument.

Reply:
that indeed seem to be the case.  I'll rip it out - better not invoke the node api when it's not needed.